contracts/DAORegistryScheme.sol

```diff
@@ -0,0 +1,181 @@
+pragma solidity >=0.5.4 <0.6.0;
+
+import "@daostack/arc/contracts/universalSchemes/UniversalScheme.sol";
+import "@daostack/arc/contracts/votingMachines/VotingMachineCallbacks.sol";
+import "@daostack/infra/contracts/votingMachines/IntVoteInterface.sol";
+import "@daostack/infra/contracts/votingMachines/VotingMachineCallbacksInterface.sol";
+import "./DAORegistry.sol";
+
+/**
+ * @title A universal scheme for letting organizations manage a registrar of named DAOs
```

Comment:
As far as I can see, there is nothing in this contract that uses the `Avatar` interface for the address that is doing the registring. (It is always cast as `address(_avatar)`). So this looks like it is more general: it can be used for DAOs to manage a registrar, but also by any other address...